docs/typography/examples/types-page.tsx

```diff
@@ -34,8 +34,14 @@ const TYPE_CARD_REGULAR_MERRIWEATHER_PROPS = {
 };
 
 const TypesPage: React.FC = () => {
+  const { palette } = useTheme();
   return (
-    <Grid container sx={{ bgcolor: '#323232' }}>
+    <Grid
+      container
+      sx={{
+        bgcolor: `${palette.mode === 'dark' ? palette.background.paper : palette.logo}`,

```

Comment:
Here again, correct me if I am wrong, but no need to string building notation.

Reply:
I've removed the `${}` and used directly the value from palette.